drumkit/README.md

```diff
@@ -0,0 +1,2 @@
+# drumkit
+A fun project made with HTML, CSS and JavaScript to implement Drum Kit.
```

Comment:
Please add a screenshot or a screen recording of your snippet in action here.

Reply:
Ok, will make the necessary changes.